mep/settings.py

```diff
@@ -204,6 +205,30 @@
 # username for logging activity by local scripts
 SCRIPT_USERNAME = 'script'
 
+# django-csp configuration for content security policy definition and
+# violation reporting - https://github.com/mozilla/django-csp
+
+# fallback for all protocols: block it
+CSP_DEFAULT_SRC = "'none'"
+
+# allow loading js locally and from google (for analytics)
+CSP_SCRIPT_SRC = ("'self'", 'https://www.googletagmanager.com')
+
+# allow loading fonts locally only
+CSP_FONT_SRC = ("'self'",)
+
+# allow loading css locally only
+CSP_STYLE_SRC = ("'self'",)
+
+# allow loading local images and google tracking pixel
+CSP_IMG_SRC = ("'self'", 'https://www.google-analytics.com')
+
+# exclude admin and cms urls from csp directives since they're authenticated
+CSP_EXCLUDE_URL_PREFIXES = ('/admin', '/cms')
```

Comment:
Does this comment mean we should also temporarily exclude the incomplete public site urls we'll be restricting for a while? Or is it ok to leave the off the list?